AutoRest/Generators/CSharp/Azure.CSharp/AzureCSharpCodeGenerator.cs

```diff
@@ -25,7 +25,8 @@ public class AzureCSharpCodeGenerator : CSharpCodeGenerator
 
         public AzureCSharpCodeGenerator(Settings settings) : base(settings)
         {
-            _namer = new AzureCSharpCodeNamer();
+
```

Comment:
Fixed!

From: Amar Zavery [mailto:notifications@github.com]
Sent: Monday, March 14, 2016 2:14 PM
To: Azure/autorest <autorest@noreply.github.com>
Cc: Ben Goldsmith <begoldsm@microsoft.com>
Subject: Re: [autorest] Change to the Azure code generator to pass settings (#841)


In AutoRest/Generators/CSharp/Azure.CSharp/AzureCSharpCodeGenerator.cs<https://github.com/Azure/autorest/pull/841#discussion_r56077116>:

> @@ -25,7 +25,8 @@ public class AzureCSharpCodeGenerator : CSharpCodeGenerator

>

>          public AzureCSharpCodeGenerator(Settings settings) : base(settings)

>          {

> -            _namer = new AzureCSharpCodeNamer();

> +

Why an empty line

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/autorest/pull/841/files#r56077116>.